build-tools/xaprepare/xaprepare/Application/Context.cs

```diff
@@ -18,8 +18,17 @@ partial class Context : AppObject
 		public const ConsoleColor FailureColor                = ConsoleColor.Red;
 		public const ConsoleColor WarningColor                = ConsoleColor.Yellow;
 
-		static readonly string XASolutionFilePath             = Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android.sln");
-		static readonly string XATestsSolutionFilePath        = Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android-Tests.sln");
+		static readonly IEnumerable<string> XASolutionFilePath = new string []
+		{
+			Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android.BootstrapTasks.sln"),
+			Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android.Build.Tasks.sln"),
+			Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android.sln"),
+		};
+
+		static readonly IEnumerable<string> XATestsSolutionFilePath = new string []
+		{
+			Path.Combine (BuildPaths.XamarinAndroidSourceRoot, "Xamarin.Android-Tests.sln"),
+		};
```

Comment:
You might put an `s` on the end of these names now, since it is an array?

Reply:
Fixed!